php/Terminus/Models/Environment.php

```diff
@@ -648,9 +648,23 @@ public function sendCommandViaSsh($command)
             'ssh -T %s@%s -p %s -o "AddressFamily inet" %s',
             [$sftp['username'], $sftp['host'], $sftp['port'], escapeshellarg($command),]
         );
+
+        // Catch Terminus running in test mode
+        $test_mode = isset($_ENV["TERMINUS_TEST_MODE"])
+            && filter_var($_ENV["TERMINUS_TEST_MODE"], FILTER_VALIDATE_BOOLEAN);
+        if ($test_mode) {
```

Comment:
Yes. All these env vars with `TERMINUS_` prefixed should be available via config.
`$this->config()->get('test_mode')`